Library/Formula/freealut.rb

```diff
@@ -36,3 +36,4 @@ def install
  
  ################################################################################
  # Checks for header files.
+ 
```

Comment:
Sorry, my editor wasn't correctly configured for whitespaces, everything should be fine now.